ansible-ipi-install/inventory/hosts.sample

```diff
@@ -67,7 +67,8 @@ dnsvip=""
 #clusterosimage=""
 # A copy of your pullsecret from https://cloud.redhat.com/openshift/install/metal/user-provisioned
 pullsecret=""
-
+# Set this variable to true to specify deployment uses IPv6 addresses
+ipv6deployment=false
```

Comment:
My suggestions below.

Create a variable labeled "ipv6_enabled" inside of roles/node-prep/defaults/main.yml and set it to false
```
ipv6_enabled: false
```
This basically ensures default is always false.

Within this hosts.sample set:
```
# Set this variable to enable IPv6 addressing
#ipv6_enabled=True
```
The capital T is important here because via the inventory/hosts file its cap sensitive. We handle catching lowercase in the 10_validation script later.

Reply:
I see you re-pushed up but didn't make the comment and variable change in the hosts.sample file. Can you fix that too?

Reply:
Aw, missed to add. Updated. Thanks